DEFED: Integrate Angular Material within the shared components module

RamiMustaklem opened the pull request:
Import Angular Material components to their own module each and export them as shared components from the components module so that these material components are encapsulated and used properly.

RamiMustaklem committed:
Dynamic Easy Front-End Dashboard:

- Changes:
 - Import default material theme to style.scss
 - Import material icons to index.html

yazaabed reviewed (commented):
The indigo-theme is the default theme?

RamiMustaklem commented:
Yes, you have to import one of the provided themes to style elements by default, and by default I mean it's the one used on the material website.

RamiMustaklem committed:
Dynamic Easy Front-End Dashboard:

- Changes:
 - Create default DEFED toolbar
 - Create DEFED toolbars module to wrap all toolbars

RamiMustaklem committed:
Dynamic Easy Front-End Dashboard:

- Changes:
 - Create default DEFED navbar
 - Create DEFED navbar module to wrap all navbars

RamiMustaklem committed:
Dynamic Easy Front-End Dashboard:

- Changes:
 - DEFED toolbar

RamiMustaklem committed:
Dynamic Easy Front-End Dashboard:

- Changes:
 - Separate DEFED toolbar component in a separate folder
 - Update DEFED toolbar component location urls
